x-pack/plugins/monitoring/public/directives/alerts/index.js

```diff
@@ -96,14 +96,19 @@ uiModule.directive('monitoringClusterAlertsListing', kbnUrl => {
 
       scope.$watch('alerts', (alerts = []) => {
         const alertsTable = (
-          <MonitoringTable
-            className="alertsTable"
-            rows={alerts}
-            placeholder="Filter Alerts..."
-            filterFields={filterFields}
-            columns={columns}
-            rowComponent={alertRowFactory(scope, kbnUrl)}
-          />
+          <Fragment>
+            <p className="kuiText kuiVerticalRhythm" tabIndex="0">Showing {alerts.length} alert(s)</p>
```

Comment:
Unsure if this is the way we want to show this or if it's in the right place, but I assumed this change would be short lived as we'll be switching this page to EUI soon.

Reply:
If it's going to be that short lived, then I'd rather we focus on changing it to use EUI instead of this.

Reply:
Yeah, I EUI will address the issue that introduced this, let's just bring in the fix when the monitoring table component is updated to use EUI.